tsconfig.build.json

```diff
@@ -2,8 +2,8 @@
   "extends": "./tsconfig.json",
   "compilerOptions": {
     "outDir": "out-tsc/lib",
-    "target": "es2015",
-    "module": "es2015",
+    "target": "es2022",
+    "module": "es2022",
```

Comment:
Warning, this is the build target that introduces the typescript bug.
This could cause issues with selectors, where the actual selector (not the value) comes through as `undefined`.
Just so that you are aware if anything strange comes up.